db/models.py

```diff
@@ -1 +1,40 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        on_delete=models.CASCADE,
+        related_name="skill_set"
```

Comment:
you set a default related_name, so it doesn't make any sense.
If you didn't add related_name parameter, then you can access skills of any race using
race.skill_set.all(). However you set the same related_name, this is why it doesn't make any sense.
It'd be better to set something like "skills", then you can access them using:
race.skills.all(). This is the purpose of related_name parameter

Reply:
Yes, you're right, in this case I use "set" as "box" and didn't think about the default name. (but using set is not so easy, I tried it in another task and it didn't work, only with related name worked well)

Reply:
Yes you're right, in this case I use "set" as "box" and didn't think about the default name. i can't change this related name becouse in tests use skill_set. 
(but using set is not so easy, I tried it in another task and it didn't work, only with related name worked well, it's kind of magic)